### PR TITLE
Add hints of programing exam and tips of input

### PR DESCRIPTION
**Pull request by kiol1812**
folder name and architecture need to think about.

### CODE REVIEW

**Review by LTurret — APPROVED** [review]
Dude, you're 🔥, thanks a lot🙏

File: Hints/0717.md
Comment (kiol1812) [inline Hints/0717.md:1]: latex didn't display will.

Comment (LTurret) [reply Hints/0717.md:1]: I'll check and fix that asap

File: programing_tips/readme.md
Comment (kiol1812) [inline programing_tips/readme.md:1]: other tips will be added?
I didn't have idea for tips which is needed.

Comment (LTurret) [reply programing_tips/readme.md:1]: Yes, I'll add tips for every challenge.
You can leave them to me 💪.

**Review by kiol1812 — COMMENTED** [review]
Some problem need to solve. I already write comment for each file which need to change.
If you saw other problem, you can just change directly.

**Pull request merged**